third_party/2/requests/api.pyi

```diff
@@ -1,11 +1,11 @@
 # Stubs for requests.api (Python 3)
 
-from typing import Union
+from typing import Union, Optional
 
 from .models import Response
 
 def request(method: str, url: str, **kwargs) -> Response: ...
-def get(url: Union[str, unicode], **kwargs) -> Response: ...
+def get(url: Union[str, unicode], params: Optional[dict]=None, **kwargs) -> Response: ...
```

Comment:
- The docstring suggests that `params` could also be a `bytes`.
- Could you be more specific about the key and value types of the dict? Even`Dict[str, Any]` would be an improvement.